### PR TITLE
persist mongo data between system restarts (don't use /tmp)

### LINKED ISSUE #1730: docker instructions don't persist mongo
the docker instructions ( https://docs.bigchaindb.com/projects/server/en/latest/appendices/run-with-docker.html ) set the mongo data volumes to persist in `/tmp` which will be cleared on reboot.  

image

this is inconsistent with how rethinkdb is configured, which saves it in $HOME/data

image

### PR DESCRIPTION
**Pull request by carchrae**
fixes #1730

### CODE REVIEW

**Comment by ttmc** [comment]
Hi @carchrae 

Sorry I didn't take a look at this pull request earlier. I was away on vacation. I see that you already agreed to the BigchainDB CLA. I will send you the CLA follow-up email now.

**Comment by krish7919** [comment]
Looks good.
Thanks, @carchrae !

File: docs/server/source/appendices/run-with-docker.md [hunk 1]
```diff
@@ -104,8 +104,8 @@ docker run \
   --name=mongodb \
   --publish=172.17.0.1:27017:27017 \
   --restart=always \
-  --volume=/tmp/mongodb_docker/db:/data/db \
-  --volume=/tmp/mongodb_docker/configdb:/data/configdb \
+  --volume=$HOME/mongodb_docker/db:/data/db \
+  --volume=$HOME/mongodb_docker/configdb:/data/configdb \
   mongo:3.4.1 --replSet=bigchain-rs
```
Comment (krish7919) [inline docs/server/source/appendices/run-with-docker.md]: Once you accept the CLA, we will be good to merge!

**Comment by carchrae** [comment]
This is to confirm that I agreed to and accepted the BigchainDB Entity (Organization) Contributor Agreement at https://www.bigchaindb.com/cla/ and to represent and warrant that I have authority to do so.

X9bqaKlBUhRq7YmmaR8AkgATBfeoA5U1atuVmNW7pJsl93mSKiSQMSpUfDcTWCe

**Comment by ttmc** [comment]
Thanks @carchrae . That completes the CLA agreement process.

@krish7919 You can accept the requested changes and merge this pull request.

**Comment by carchrae** [comment]
happy to help @ttmc - we are just starting to use bigchaindb so will try to contribute where we can.

**Pull request merged**